drivers/include/sx126x.h

```diff
@@ -97,6 +99,23 @@ void sx126x_setup(sx126x_t *dev, const sx126x_params_t *params, uint8_t index);
  */
 int sx126x_init(sx126x_t *dev);
 
+/**
+ * @brief   Converts symbol value to time in milliseconds.
+ *
+ * @param[in] dev                      Device descriptor of the driver
+ * @param[in] symbols                  Symbols
+ *
+ * @return Time for symbol(s) in milliseconds
+ */
+static inline int sx126x_symbol_to_msec(sx126x_t *dev, uint16_t symbols)
+{
+    assert(dev && (dev->mod_params.bw <= SX126X_LORA_BW_500) && \
+           (dev->mod_params.bw >= SX126X_LORA_BW_125));
+
+    /* Refer section 4.1.1.7  Time on air in SX1276/77/78/79 datasheet */
```

Comment:
```suggestion
    /* Refer section 4.1.1.7  Time on air in SX1276/77/78/79 datasheet */
```

Seems like its the wrong datasheet reference...

Reply:
I can't find the section either

Reply:
![1](https://user-images.githubusercontent.com/5170658/124893935-79d89100-dfdb-11eb-9308-af8d0a2c9a71.png)
![2](https://user-images.githubusercontent.com/5170658/124893972-82c96280-dfdb-11eb-8401-050ada0c6ad1.png)

Reply:
In [here](https://cdn-shop.adafruit.com/product-files/3179/sx1276_77_78_79.pdf)

Reply:
Yeah well its the wrong datasheet, this is sx126x not 127x, I find it weird to have a reference to another driver datasheet.

Reply:
Can the correct datasheet be referenced?

Reply:
It was more about the math. Let me see if there are references in the sx126x datasheet.

Reply:
Updated.